Classes/commands.py

```diff
@@ -22,6 +24,7 @@
 from Classes.menus import Confirm
 import Classes.errors as errors
 import Classes.checks as checks
+from Classes.extra_functions import get_settings_file, role_id_index
```

Comment:
Here you imported get_settings_file but I don't see where it is used in this file or why it should be used in this file. If it's an old reference just go ahead and remove it.

Reply:
Fixed in commit